src/cargo/core/compiler/lto.rs

```diff
@@ -14,6 +15,8 @@ use std::collections::hash_map::{Entry, HashMap};
 /// `OnlyBitcode`      | `-C linker-plugin-lto` |             | ✓
 /// `ObjectAndBitcode` |                        | ✓           | ✓
 /// `OnlyObject`       | `-C embed-bitcode=no`  | ✓           |
+///
+/// See <https://doc.rust-lang.org/nightly/cargo/reference/profiles.html#lto>
```

Comment:
Thanks. However, this doesn't look good in rustdoc.

<img width="500" alt="image" src="https://user-images.githubusercontent.com/14314532/218257275-06835d36-9739-4e40-9e5e-3003b57a169b.png">

To maintain readability in both rustdoc and raw content, instead of adding "LTO: Link Time Optimizations", I suggest simply give a link on LTO text, such as

```diff
- /// LTO: Link Time Optimizations
- /// Possible ways to run rustc and request various parts of LTO.
+ /// Possible ways to run rustc and request various parts of [LTO].
  /// ...
  /// ...
- /// See <https://doc.rust-lang.org/nightly/cargo/reference/profiles.html#lto>
+ /// [LTO]: https://doc.rust-lang.org/nightly/cargo/reference/profiles.html#lto
```

Reply:
`cargo doc --document-private-items --no-deps --open` and search lto.

BTW, could you help squash this pull request to one commit to keep the commit history clean?

Reply:
You can reset to the latest commit on master branch and add your patch, as this patch is quite small. For example, `git reset --hard origin/master`



[git reset tutorial](https://www.atlassian.com/git/tutorials/undoing-changes/git-reset)